src/quo/components/wallet/account_card/properties.cljs

```diff
@@ -3,17 +3,13 @@
 
 (defn gradient-start-color
   [theme customization-color]
-  (colors/theme-colors
-   (colors/resolve-color customization-color theme 0)
-   colors/neutral-95
-   theme))
+  (colors/resolve-color customization-color theme 0))
 
 (defn gradient-end-color
   [theme customization-color]
-  (colors/theme-colors
-   (colors/resolve-color customization-color theme 6)
-   colors/neutral-95
-   theme))
+  (colors/theme-colors (colors/resolve-color customization-color theme 6)
```

Comment:
resolve colors handles theming anyway.
`(colors/resolve-color customization-color theme 6)` - is all you need

Reply:
Thanks @J-Son89